pkg/images/images.go

```diff
@@ -31,25 +33,22 @@ type CustomImage struct {
 	DockerfileOptions []string `yaml:"dockerfileOptions"`
 }
 
-// Images is the data structure that will hold all image definitions.
-var Images []Image
-
 func ImageName(organisation string, image string) string {
```

Comment:
Not really part about this PR but still something:
IMO this should be called `Name` - the package is already called `Image` - `images.ImageName` stutters.